client/ayon_houdini/plugins/publish/validate_frame_token.py

```diff
@@ -31,8 +31,8 @@ def process(self, instance):
 
         invalid = self.get_invalid(instance)
         if invalid:
-            raise RuntimeError(
-                "Output settings do no match for '%s'" % instance
+            raise PublishError(
+                f"Output settings do no match for '{instance}'" 
```

Comment:
Should I used `PublishValidationError` instead here? 
```suggestion
            raise PublishValidationError(
                f"Output settings do no match for '{instance}'",
                title=self.label
```

Reply:
I think that may be better **if** the user can do something about it. Just seeing "Output setting do not match" I have no idea what to do next. What output settings mismatch? What can I do about it? Why do they mismatch, we should inform the artist what is going on and what to do about it.

Reply:
any update on this comment? I agree with Roy that the message needs more clarification.